storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -52,64 +47,56 @@ message:
 
 .. warning:: 
 
-   |xtrabackup| will not copy the keyring file into the backup directory. In
-   order to prepare the backup, you must make a copy of the keyring file yourself.
+   |xtrabackup| does not copy the keyring file into the backup directory. To prepare the backup, you must copy the keyring file manually.
 
 .. rubric:: Preparing the Backup
 
-In order to prepare the backup you need to specify the keyring-file-data
-(server-id is stored in the :file:`backup-my.cnf` file, so it can be omitted when
-preparing the backup, regardless of the |MySQL| version used).
+To prepare the backup specify the keyring-file-data.
 
 .. code-block:: bash
 
    $ xtrabackup --prepare --target-dir=/data/backup \
    --keyring-file-data=/var/lib/mysql-keyring/keyring
 
-After |xtrabackup| is finished preparing the backup, you should see the
-following message:
+After |xtrabackup| takes the backup, the following
+message confirms the action:
 
 .. code-block:: bash
 
    InnoDB: Shutdown completed; log sequence number 5697064
    160401 10:34:28 completed OK!
 
 The backup is now prepared and can be restored with the :option:`--copy-back`
-option. In case the keyring has been rotated, you need to restore the keyring
-which was used to take and prepare the backup.
+option. In case the keyring has been rotated, you must restore the keyring used when the backup was  taken and prepared.
 
 Using ``keyring_vault`` plugin
 ------------------------------
 
 The support for encrypted InnoDB tablespace backups with ``keyring_vault`` has been
-implemented in |Percona XtraBackup| 2.4.11.  Keyring vault plugin settings are
+implemented in |Percona XtraBackup| 2.4.11. Keyring vault plugin settings are
```

Comment:
why do we mention 2.4 in the 8.0 documentation ?

Reply:
removed.